var/spack/repos/builtin/packages/ccache/package.py

```diff
@@ -7,16 +7,20 @@
 import re
 
 
-class Ccache(AutotoolsPackage):
+class Ccache(CMakePackage):
     """ccache is a compiler cache. It speeds up recompilation by caching
     previous compilations and detecting when the same compilation is being done
     again."""
 
     homepage = "https://ccache.samba.org/"
```

Comment:
It seems that the canonical URL is now https://ccache.dev, we might as well update to that if the package is being updated.